kod/object/item/passitem/chalice.kod

```diff
@@ -151,7 +190,7 @@ messages:
 
          Post(self,@Delete);
       }
-
+      
```

Comment:
Undo whitespace change.

Reply:
Oof, I'll have to keep an eye out for those in the future.  I blame VS~